SPECS/rust/rust.spec

```diff
@@ -28,18 +32,29 @@ Source6:        https://static.rust-lang.org/dist/%{release_date}/rustc-%{stage0
 Source7:        https://static.rust-lang.org/dist/%{release_date}/rust-std-%{stage0_version}-aarch64-unknown-linux-gnu.tar.gz
 BuildRequires:  binutils
 BuildRequires:  cmake
+# make sure rust relies on curl from CBL-Mariner (instead of using its vendored flavor)
 BuildRequires:  curl-devel
 BuildRequires:  git
 BuildRequires:  glibc
+# make sure rust relies on libgit2 from CBL-Mariner (instead of using its vendored flavor)
+BuildRequires:  libgit2-devel
```

Comment:
Question: How would this ensure that we are using this version of `libgit2` and not the one that is in our cache ?

Reply:
this is by design in rust code (see https://github.com/rust-lang/git2-rs/blob/master/libgit2-sys/build.rs)

Reply:
I also verified after installing rust that cargo uses our libgit2 (and others) using 'ldd cargo...')